thumbnail/thumbnail_magick.go

```diff
@@ -195,7 +195,7 @@ func isPNG(bytes []byte) bool {
 
 func isWEBP(bytes []byte) bool {
 	// 0x52 = R, 0x49 = I, 0x46 = F
-	if bytes[0] != 0x52 || bytes[1] != 0x49 || bytes[2] != 0x46 || bytes[2] != 0x46 {
+	if bytes[0] != 0x52 || bytes[1] != 0x49 || bytes[2] != 0x46 {
```

Comment:
Actually, the header should have 'RIFF'. So this check would be
`if bytes[0] != 0x52 || bytes[1] != 0x49 || bytes[2] != 0x46 || bytes[3] != 0x46 {`
(https://github.com/smartnews/yoya-thumber/issues/25)